Ingest data from EMQX broker (#2663)
risingwavelabs/risingwave-docs: docs/guides/grafana-integration.md

See the [GRANT](/sql/commands/sql-grant.md) command for more details.

> Now that RisingWave is added as a database, you can start creating dashboards within Grafana using the data in RisingWave. For an extensive tutorial that covers how to create dashboards in Grafana with data queried from RisingWave, check out the [Use RisingWave to monitor RisingWave metrics](/tutorials/monitor-rw-metrics.md) tutorial, which uses a demo cluster so you can easily try it out on your device.

Reviewer: ```suggestion
Now that RisingWave is added as a database, you can start creating dashboards within Grafana using the data in RisingWave. To learn about how to create dashboards in Grafana with data queried from RisingWave, check out this tutorial: [Use RisingWave to monitor RisingWave metrics](/tutorials/monitor-rw-metrics.md). It details the steps using a demo cluster, which you can try out on your device.
```

## Configure Grafana to read data from RisingWave Cloud

Grafana can also be configured to visualize and monitor data from a RisingWave Cloud cluster, allowing you to build charts and real-time dashboards based on tables and materialized views. Follow these steps to integrate RisingWave as a data source in Grafana.

### Add RisingWave Cloud as a Data Source in Grafana

> 1. **Access Data Source Settings:**

Reviewer: These steps should not be bolded. Please reword these to be proper sentences.

Reviewer: You can start the step with the sentence, but end it with a period. Basically, stating the purpose and then the actual operations is good, but they shoud be arranged as normal sentences.
